### PR TITLE
Bump cibuildwheel and 0.11.5rc

### PR DESCRIPTION
**Pull request by rnpridgeon**

### CODE REVIEW

File: confluent_kafka/src/confluent_kafka.c [hunk 1]
```diff
@@ -2156,7 +2156,7 @@ static PyObject *libversion (PyObject *self, PyObject *args) {
 }
 
 static PyObject *version (PyObject *self, PyObject *args) {
-	return Py_BuildValue("si", "0.11.5rc5", 0x000b0500);
+	return Py_BuildValue("si", "0.11.5rc6", 0x000b0500);
```
Comment (edenhill) [inline confluent_kafka/src/confluent_kafka.c]: You should set this to 0.11.5, doesn't need to be bumped for each rc

**Comment by edenhill** [comment]
It seems the osx builder isn't actually building anything, probably unrelated to this PR but it is a regression

**Comment by rnpridgeon** [comment]
Added py27 and py36 builders to the build index. It's slightly hacky but unfortunately it would seem travis ci doesn't make this easy today. 

https://github.com/travis-ci/travis-ci/issues/2312

File: .travis.yml [hunk 2]
```diff
@@ -12,6 +12,24 @@ matrix:
     dist: trusty
     python: "3.6"
     env: LD_LIBRARY_PATH="$PWD/tmp-build/lib" LIBRDKAFKA_VERSION=master
+  # Source package verification with Python 2.7 and librdkafka master
+  - os: osx
+    python: "2.7"
+    env: LD_LIBRARY_PATH="$PWD/tmp-build/lib" LIBRDKAFKA_VERSION=master
+    before_install:
+        - brew update && brew upgrade pyenv
```
Comment (edenhill) [inline .travis.yml]: nit: no reason to use && instead of separate lines

**Comment by rnpridgeon** [comment]
holding for librdkafka 0.11.5 release so we can update the version here

**Comment by rnpridgeon** [comment]
This fell out of sight and has since been fixed in a few other PRs

**Pull request closed without merging**